.github/workflows/triage.yml

```diff
@@ -0,0 +1,53 @@
+name: Triage new issues
+
+on:
+  issues:
+    types: [ opened ]
+
+env:
+  GITHUB_TOKEN: ${{ secrets.PROJECT_BOARD_AUTOMATION }}
```

Comment:
So it still needs a token? We won't get away with the built-in token for managing boards?

Reply:
Actually no, we do not need it. I tested in a private repo under a organization that I created to play with the configurations using classic projects, but now I realized that I should have used the [repository](https://docs.github.com/en/graphql/reference/objects#repository) API instead of the organization, since the current Solidity project board is in the repo scope and not in the org. I will update the query, so the org permissions will not be necessary.